test/models/common/fwdaccuracy.py

```diff
@@ -116,6 +116,7 @@ def validate_forward_accuracy(
     file_name = (
         Path(__file__).parents[1].resolve() / Path("data") / Path(file_name.lower())
     )
+    print("loaded file: ", file_name)
```

Comment:
Can we remove the added print statements?

Reply:
let me inspect all files in that folder.

Reply:
fixed